mlrun/datastore/targets.py

```diff
@@ -1470,11 +1458,26 @@ def __init__(
         create_collection: bool = False,
         override_collection: bool = False,
     ):
-
+        """
+        Write to MongoDB as output target for a flow.
+
+        :param db_name:             the name of the database to access
+        :param connection_string:   your mongodb connection string
+        :param collection_name:     the name of the collection to access,
+                                    from the current database
+        :param create_collection:   pass True if you want to create new collection named by
+                                    collection_name on current database.
+        :param override_collection: pass True if you want to override all the documents on the
+                                    collection named by collection_name on current database.
+        """
```

Comment:
superb docs 💪 !!! why can't you do these for all the ones you are writing 🥺 ?

Reply:
thank :)